### PR TITLE
PHP Push Example

### PR DESCRIPTION
**Pull request by mskian**
- PHP Push Example
- Update Some devDependencies Due to NPM Found some vulnerabilities Fixed using `npm audit fix`
- Fix Readme File Markdown Errors

### CODE REVIEW

**Comment by eternal-flame-AD** [comment]
Hi, your Travis build is failing because the markdown file is not formatted. Please run `npm run format` and commit the results.

**Comment by mskian** [comment]
@eternal-flame-AD  Done Passed

File: docs/more-pushmsg.md [hunk 1]
```diff
@@ -33,3 +33,66 @@ func main() {
...
+curl_setopt($ch, CURLOPT_URL, $url);
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch,CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch,CURLOPT_POSTFIELDS, $data_string);
+
+$result = curl_exec($ch);
+$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+if ($code == 400)
```
Comment (eternal-flame-AD) [inline docs/more-pushmsg.md]: Why not use a switch … case … for this?

File: docs/more-pushmsg.md [hunk 2]
```diff
@@ -33,3 +33,66 @@ func main() {
...
+  }
+  else
+  {
+   if ($code == 404)
+   {
+    $message = '<strong>API URL Not Found</strong>';
+   }
+   else
+   {
+    $message = '<strong>Your Message was Submitted</strong>';
```
Comment (eternal-flame-AD) [inline docs/more-pushmsg.md]: Only 200 status code should be accepted, in your implementation a 502 could also be considered success.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
--> https://gotify.net
+-> <https://gotify.net>
 
 ## Run locally
 
 ```bash
-$ (cd website && npm install)
-$ (cd website && npm run start)
+cd website && npm install
```
Comment (eternal-flame-AD) [inline README.md]: Without the brackets, the current work directory would be changed.

**Comment by eternal-flame-AD** [comment]
Next time, please try to separate independent works into different PRs. This would make review and discussion easier.

File: docs/more-pushmsg.md (hunk 2 above)
Comment (mskian) [reply docs/more-pushmsg.md]: Let me check the Official Docs clearly Give me a moments pls

**Comment by mskian** [comment]
@eternal-flame-AD sure

File: docs/more-pushmsg.md (hunk 1 above)
Comment (mskian) [reply docs/more-pushmsg.md]: Done It Validate using HTTP status code `https://gotify.net/api-docs#/message/createMessage`

```php
$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
```

File: docs/more-pushmsg.md (hunk 2 above)
Comment (mskian) [reply docs/more-pushmsg.md]: Done It Validate using HTTP status code `https://gotify.net/api-docs#/message/createMessage`

```php
$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
```

File: docs/more-pushmsg.md [hunk 4]
```diff
@@ -33,3 +33,53 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: Is there a reason for using the older array() syntax and not brackets `[]`?

File: docs/more-pushmsg.md [hunk 5]
```diff
@@ -33,3 +33,53 @@ func main() {
...
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch,CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch,CURLOPT_POSTFIELDS, $data_string);
+
+$result = curl_exec($ch);
+$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+curl_close ($ch);
+
+switch ($code) {
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: I think we can omit this, the other examples also don't include error handling.

File: docs/more-pushmsg.md [hunk 6]
```diff
@@ -33,3 +33,53 @@ func main() {
...
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array(
+    'Content-Type: application/json; charset=utf-8'
+);
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, $url);
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: could you fix the formatting here and for the other lines?
```suggestion
curl_setopt($ch, CURLOPT_HTTPHEADER, $headers );
```

File: docs/more-pushmsg.md [hunk 7]
```diff
@@ -33,3 +33,53 @@ func main() {
...
+    "message"=> "Test push From PHP cURL.",
+    "priority"=> 5,
+);
+
+$data_string = json_encode($data);
+
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array(
+    'Content-Type: application/json; charset=utf-8'
```
Comment (jmattheis) [inline docs/more-pushmsg.md]: Could you use either single or double quotes over the whole php block for consistency?

File: docs/more-pushmsg.md (hunk 5 above)
Comment (mskian) [reply docs/more-pushmsg.md]: I feel it better to understand while using this on PHP Forms

File: docs/more-pushmsg.md (hunk 4 above)
Comment (mskian) [reply docs/more-pushmsg.md]: Sorry for this it's Updated Now

**Pull request merged**